Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -1042,7 +1042,11 @@ private static void Truncate(LiveResult result, DateTime start, DateTime stop)
                 }
             }
             result.Charts = charts;
-            result.Orders = result.Orders.Values.Where(x => x.Time >= start && x.Time <= stop).ToDictionary(x => x.Id);
+            result.Orders = result.Orders.Values.Where(x =>
+                (x.Time >= start && x.Time <= stop) ||
+                (x.LastFillTime != null && x.LastFillTime >= start && x.LastFillTime <= stop) ||
+                (x.LastUpdateTime != null && x.LastUpdateTime >= start && x.LastUpdateTime <= stop)
```

Comment:
Just a note: this could potentially cause the same order to be present in different intervals of start and stop